### PR TITLE
feat: do not deduct tax in ibc token

### PR DESCRIPTION
**Pull request by nowooj**
## Summary of changes

The policy has been changed to not deduct tax for IBC tokens.

## Report of required housekeeping

- [ ] Github issue OR spec proposal link
- [x] Wrote tests
- [x] Added a relevant changelog entry to CHANGELOG.md

----

## (FOR ADMIN) Before merging

- [ ] Added appropriate labels to PR
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" (coding standards)
- [ ] Confirm added tests are consistent with the intended behavior of changes
- [ ] Ensure all tests pass

### CODE REVIEW

File: packages/classic_terraswap/src/asset.rs [hunk 1]
```diff
@@ -10,6 +11,8 @@ use cosmwasm_std::{
 };
 use cw20::Cw20ExecuteMsg;
 
+pub static REX: Lazy<Regex> = lazy_regex!("^ibc/[A-F0-9]{64}$");
```
Comment (jbamlee) [inline packages/classic_terraswap/src/asset.rs:14]: nit.
Clear naming like ibc_rex

**Review by jbamlee — DISMISSED** [review]
Question. 
Does IBC token have only A-F characters?

**Comment by JoowonYun** [comment]
> Question. Does IBC token have only A-F characters?

IBC hash is SHA256. So, it is a 64-digit hexadecimal hex value.

File: packages/classic_terraswap/src/asset.rs (hunk 1 above)
Comment (JoowonYun) [reply packages/classic_terraswap/src/asset.rs]: https://github.com/terraswap/classic-terraswap/pull/16/commits/818a2b27179554ae7801be2a721f08b5b256939c

**Review by psy2848048 — APPROVED** [review]

**Review by jhlee-young — APPROVED** [review]

**Pull request merged**